book/index.qmd

```diff
@@ -1,24 +1,37 @@
 ## Preface {.unnumbered}
 
-This website hosts the course materials for the [Animals in Motion](https://neuroinformatics.dev/open-software-week/animals-in-motion.html) workshop,
-part of the inauguaral [Neuroinformatics Unit Open Software Week](https://neuroinformatics.dev/open-software-week/index.html).
+**Animals in Motion** is an online handbook designed for researchers and students interested in learning about free open-source tools for tracking animal motion from video footage and extracting quantitative descriptions of behaviour from motion tracks.
```

Comment:
```suggestion
**Animals in Motion** is an online handbook that introduces free open-source tools for tracking animal motion from videos and extracting quantitative descriptions of behaviour from motion tracks.
```

broader audience?

Reply:
Love this! It also makes the sentence much shorter.